tests/blackbox/stratisd_cert.py

```diff
@@ -90,8 +90,8 @@ def test_get_managed_objects(self):
             [
                 "busctl",
                 "call",
-                "org.storage.stratis1",
-                "/org/storage/stratis1",
+                "org.storage.stratis2",
```

Comment:
Please use constants defined in testlib/dbus.py here, instead of literal strings and make them public.

Reply:
We realized that we have a fine method for doing this already in testlib/dbus.py. So we'll call that method, instead, and these string literals will just go away as a result.

Reply:
I've switched `_get_managed_objects()` to `get_managed_objects()` and am using that instead of busctl. Resolved.